website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -22,42 +22,36 @@ Admin partitions exist a level above namespaces in the identity hierarchy. They
 
 ### Default Admin Partition
 
-Each Consul cluster will have at least one default admin partition (named `default`). Any resource created without specifying an admin partition will inherit the partition of the ACL token.
+Each Consul cluster will have a default admin partition (named `default`). The `default` admin partition is special because it may contain namespaces and other entities that are replicated between datacenters. The `default` partition must also contain the Consul servers.  
```

Comment:
```suggestion
Each Consul cluster will have a default admin partition (named `default`). The `default` admin partition is special because it can contain namespaces and other resources that are replicated between datacenters. The `default` partition must also contain the Consul servers.  
```